app/src/pages/inside/logsPage/logItemInfo/defectDetails/defectDetails.jsx

```diff
@@ -90,26 +90,23 @@ const messages = defineMessages({
     defaultMessage: 'Comment',
   },
 });
+const MAKE_DECISION = 'make_decision';
+
 const POST_ISSUE_EVENTS_INFO = {
-  postBtn: LOG_PAGE_EVENTS.POST_ISSUE_MODAL_EVENTS.POST_BTN_POST_ISSUE_MODAL,
-  attachmentsSwitcher: LOG_PAGE_EVENTS.POST_ISSUE_MODAL_EVENTS.attachmentsSwitcher,
-  logsSwitcher: LOG_PAGE_EVENTS.POST_ISSUE_MODAL_EVENTS.logsSwitcher,
-  commentSwitcher: LOG_PAGE_EVENTS.POST_ISSUE_MODAL_EVENTS.commentSwitcher,
+  postBtn: LOG_PAGE_EVENTS.POST_ISSUE_MODAL_EVENTS.POST_BTN_POST_ISSUE_MODAL(MAKE_DECISION),
```

Comment:
Please use `camelCase` for functions.
As I see these event objects used for regular `post/link` issue actions too, not only for called from make decision.
So please consider this case.